File: examples/TemplateDeleteExample.rb
require "json"
require "dropbox-sign"

Dropbox::Sign.configure do |config|
    config.username = "YOUR_API_KEY"
    # config.access_token = "YOUR_ACCESS_TOKEN"
end

begin
    Dropbox::Sign::TemplateApi.new.template_delete(
        "f57db65d3f933b5316d398057a36176831451a35", # template_id
    )
rescue Dropbox::Sign::ApiError => e
    puts "Exception when calling TemplateApi#template_delete: #{e}"
end

File: sandbox/dotnet/src/Dropbox.SignSandbox/TemplateDeleteExample.cs
using System;
using System.Collections.Generic;
using System.IO;
using System.Text.Json;

using Dropbox.Sign.Api;
using Dropbox.Sign.Client;
using Dropbox.Sign.Model;

namespace Dropbox.SignSandbox;

public class TemplateDeleteExample
{
    public static void Run()
    {
        var config = new Configuration();
        config.Username = "YOUR_API_KEY";
        // config.AccessToken = "YOUR_ACCESS_TOKEN";

        try
        {
            new TemplateApi(config).TemplateDelete(
                templateId: "f57db65d3f933b5316d398057a36176831451a35"
            );
        }
        catch (ApiException e)
        {
            Console.WriteLine("Exception when calling TemplateApi#TemplateDelete: " + e.Message);
            Console.WriteLine("Status Code: " + e.ErrorCode);
            Console.WriteLine(e.StackTrace);
        }
    }
}
